cfme/tests/intelligence/reports/test_metering_report.py

```diff
@@ -283,7 +287,7 @@ def test_validate_cpu_usage(resource_usage, metering_report):
 @pytest.mark.uncollectif(
     lambda provider: provider.one_of(EC2Provider) or provider.one_of(GCEProvider))
```

Comment:
**optional** while we are at it, an chance for a unrelated enhancement `one_of` takes arbitrary arguments, and the expression should read more nice if it is condensed to one method call